src/firefly/js/charts/ChartsCntlr.js

```diff
@@ -354,7 +355,7 @@ function chartUpdate(action) {
 
 
         const {data, fireflyData} = Object.entries(changes)
-                             .filter(([k,v]) => (k.startsWith('data') || k.startsWith('fireflyData')))
+                             .filter(([k,]) => (k.startsWith('data') || k.startsWith('fireflyData')))
```

Comment:
`([k,])` this is great.  This is good compromise between unused variable warning and an indication that more parameters are available for use if needed.

Reply:
In the latest commit:
- Fixed filtering issues for expressions with alpha-numeric columns. Unfortunately, if a column name contains space, filter will not work the second time, because filter parser is using space as a delimiter. You can check it on a NED table in irsawebdev1.
- Updated histogram request to get the expression calculation in the database.
- Fixed selection bug in heatmap I have introduced when checking that user is selecting active trace.

I can not reproduce column dialog issue reliably.

Not sure, I can move lg->log10 to DbAdopter at the moment, I need to get Expression parser to handle quoted tokens, to make sure "lg(" is not part of the column name. 

I'll create tickets for outstanding issues.